### DISCUSSION TITLE
Adding more LoRa settings/Trigger Words

### DISCUSSION THREAD

**Discussion by Nelayme** [lllyasviel/Fooocus#3096] [Ideas]
Hi,

I've downloaded over 20 different LoRas, and more than half of them work best once a trigger word is used. However, it's impossible to remember all the trigger words for each Lora, and it's a waste of time to open Civitai every time I need to use one.

Wouldn't it be better to have a quick setting just like in Forge/A1111 where you can check the trigger words instead?

**Comment by mashb1t** [comment]
This is already in the backlog, but currently not planned:
- https://github.com/lllyasviel/Fooocus/issues/1476
- https://github.com/lllyasviel/Fooocus/issues/2029

**Comment by ILikeToasters** [comment]
This is one of the reasons I started using Stability Matrix to manage my models. I can go there and all my LoRAs are listed and I can just copy from there. Not perfect but better than any other way plus it works well for managing models.
